handlers/wallet_service.py

```diff
@@ -0,0 +1,118 @@
+import json
+from typing import Optional, Union
+
+from aws_lambda_context import LambdaContext
+
+from common.errors import ApiError, RdsNotFoundError
+from usecases.wallet_service import WalletService
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def handle_address_balance(
+    event: ApiGatewayEvent,
+    __: LambdaContext,
+    wallet_service: Union[WalletService, None] = None
+) -> dict:
+
+    wallet_service = wallet_service or WalletService()
+
+    address: Optional[str] = event.query.get("address")
+    token: Optional[str] = event.query.get("token")
+
+    if address is None or token is None:
+        raise ApiError("invalid_parameters")
+
+    response: Optional[dict]
+    try:
+        response = wallet_service.address_balance(address, token)
+    except RdsNotFoundError:
```

Comment:
I saw that we also raise `RdsError('only one row expected')`. Should we also translate that to an `ApiError`?

Reply:
Since the query itself limits the response to 1 row it should never happen, and if it does it is an error that should be alerted.
Letting it raise the exception and returning an 500 seems like the correct option in this case.